src/server/index.js

```diff
@@ -11,14 +11,22 @@ dotenv.config();
 let port = process.env.PORT || 5500;
 const mongoURI = process.env.MONGO_URI;
 
+// console.log("MONGODB URL ",mongoURI);
 const app = express();
-
+const router = Router();
+const Userroute = require("./Routes/User");
+const AuthRoute = require("./Routes/Auth");
 app.use(cors());
 app.use(express.json());
 
 app.get("/", (req, res)=>{
     res.send("Quikz: Hello World!");
 });
+app.use(express.json()); // to parse application/json
+
+app.use('/api/users', Userroute);
```

Comment:
This should have been in different file dedicated for routing. We cannot clutter everything in index.js file. Difficult to maintain and scale.